### ISSUE TITLE
Fails on simple inputs

### ISSUE THREAD

**Issue by avangogo** [hirsch-lab/cyminiball#2]
The main function occasionally on some inputs. Here is an example.
```
a = np.array([
[-1.3853526520724133,0.3378483290927974,-1.3981381087490836,-1.3129767272601454],
[-1.5065205225160652,0.3378483290927974,-1.3412724047598314,-1.3129767272601454],
[-1.3853526520724133,0.3378483290927974,-1.2275409967813267,-1.3129767272601454]])
    miniball.compute(a)
```
I obtain `miniball.MiniballError: Encountered a problem when computing the miniball.`
I don't see why this input would be problematic.

**Comment by normanius** [comment]
Yes, I encountered this as well. I was about to release a new version where this is fixed. Stay tuned.

The problem is caused by a failing numerical integrity check. [This check](https://github.com/hirsch-lab/cyminiball/blob/dc68c34d767e6ed992c9d5b8614731b59253752a/miniball/miniball.hpp#L148) currently is too sensitive for the Cython-generated code (the C++ code works without troubles).

**Comment by normanius** [comment]
@avangogo I fixed the issue quite quickly. But it took me a bit longer than expected to set up some other things on my bucket list, e.g. some CI infrastructure. I currently face some issues on Windows.

Until I'm able to release the update on PyPI, you can try to build the package on Linux or MacOS with the following instrcutions.

```bash
git clone https://github.com/hirsch-lab/cyminiball.git
cd cyminiball
./deploy.sh
```

Note that I've change the package name (to match the project name: cyminiball), so you have to `import cyminiball as mb`. Hope this works for you

**Comment by normanius** [comment]
This issue is fixed with version v2.0.0. Fetch the new version via `pip install --upgrade cyminiball`

**Comment by avangogo** [comment]
Thanks for your reactivity! Works well for me.
